pkg/sources/chunker.go

```diff
@@ -26,20 +26,24 @@ func Chunker(originalChunk *Chunk) chan *Chunk {
 		r := bytes.NewReader(originalChunk.Data)
 		reader := bufio.NewReaderSize(bufio.NewReader(r), ChunkSize)
 		for {
-			chunkBytes := make([]byte, ChunkSize)
+			chunkBytes := make([]byte, ChunkSize+PeekSize)
 			chunk := *originalChunk
+			chunkBytes = chunkBytes[:ChunkSize]
```

Comment:
I wonder if the performance would look like with just these changes.

`append` won't need to reallocate if there's enough capacity in the slice.

Reply:
Would it make sense to have the total `chunkBytes` + `peekBytes` have a different var name to avoid confusion?

Reply:
- append vs copy : very similar.
![Screenshot 2023-07-24 at 11 37 38 AM](https://github.com/trufflesecurity/trufflehog/assets/21311841/4865d085-7318-4136-9da8-6b8be9d8a7bb)